microsoft-edge/webview2/release-notes/index.md

```diff
@@ -63,6 +63,310 @@ No additional APIs have been promoted to Stable and added in this Release SDK.
+  * [ICoreWebView2ExperimentalFrame8::add_FrameCreated](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalframe8?view=webview2-1.0.3079-prerelease&preserve-view=true#add_framecreated)
+  * [ICoreWebView2ExperimentalFrame8::remove_FrameCreated](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalframe8?view=webview2-1.0.3079-prerelease&preserve-view=true#remove_framecreated)
+<!-- todo: add Stop Property, like WinRT has?  event, or property? -->
+
+* [ICoreWebView2ExperimentalFrameChildFrameCreatedEventHandler](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalframechildframecreatedeventhandler?view=webview2-1.0.3079-prerelease&preserve-view=true)
+
+---
+
+
+<!-- ---------- -->
+###### Show WPF elements on top of the WebView2 layer (WebView2CompositionControl)
+
+The `WebView2CompositionControl` prevents the WebView2 control from being the topmost layer in a WPF app and obscuring any WPF elements.  `Microsoft.Web.WebView2.Wpf.WebView2CompositionControl` is a drop-in replacement for the standard WPF WebView2 control.  Both the WebView2 control and `WebView2CompositionControl` implement the `Microsoft.Web.WebView2.Wpf.IWebView2` interface.  Both of them derive from `FrameworkElement`, as follows:
+* `FrameworkElement` -> `HwndHost` -> `WebView2`.
+* `FrameworkElement` -> `Control` -> `WebView2CompositionControl`.
+
+Background: If you're building a Windows Presentation Foundation (WPF) app and using the WebView2 control, you may find that your app runs into "airspace" issues, where the WebView2 control is always displayed on top, hiding any WPF elements in the same location, even if you try to specify the WPF elements to be above the WebView2 control (using visual tree order or the z-index property, for example).
+
+This issue occurs because the WPF control uses the WPF `HwndHost` to host the Win32 WebView2 control, and `HwndHost` has an issue with airspace; see [Mitigating Airspace Issues In WPF Applications](https://dwayneneed.github.io/wpf/2013/02/26/mitigating-airspace-issues-in-wpf-applications.html).
+<!-- todo: [PR 4804: WPF Airspace - WebView2CompositionControl](https://github.com/MicrosoftEdge/WebView2Feedback/pull/4804/files) -->
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2CompositionController` Class
+<!-- todo: add new members -->
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2 Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.3079-prerelease&preserve-view=true)<!-- todo: delete link to large old class -->
+<!-- todo: add new members? -->
+
+* [CoreWebView2CompositionController Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2compositioncontroller?view=webview2-winrt-1.0.3079-prerelease&preserve-view=true)
+<!-- todo: add new members -->
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalCompositionController5]<!-- todo: link -->
```

Comment:
link to ICoreWebView2ExperimentalCompositionController5 (guessing # 5 not 6), and link to its members.  then propagate list to other two tabs.

open PR for Win32 API Ref:
https://github.com/MicrosoftDocs/webview2-win32-reference/pull/108

for WinRT:
https://github.com/MicrosoftDocs/webview2-winrt-reference/pull/77

@sivMSFT @victorhuangwq

Reply:
comment n/a, b/c this section is WebView2CompositionControl not CoreWebView2CompositionController.